files/build_templates/swss_vars.j2

```diff
@@ -11,5 +11,6 @@
     "supporting_bulk_counter_groups": "{{ DEVICE_METADATA.localhost.supporting_bulk_counter_groups|join(',') }}",
     {% endif -%}
     "dual_tor": {% if DEVICE_METADATA.localhost.type == "ToRRouter" and DEVICE_METADATA.localhost.subtype == "DualToR" %}"enable"{% else %}"disable"{% endif %},
-    "dscp_remapping": {% if SYSTEM_DEFAULTS is defined and SYSTEM_DEFAULTS.tunnel_qos_remap is defined and SYSTEM_DEFAULTS.tunnel_qos_remap.status == "enabled" %}"enable"{% else %}"disable"{% endif %}
+    "dscp_remapping": {% if SYSTEM_DEFAULTS is defined and SYSTEM_DEFAULTS.tunnel_qos_remap is defined and SYSTEM_DEFAULTS.tunnel_qos_remap.status == "enabled" %}"enable"{% else %}"disable"{% endif %},
+    "type": "{{ DEVICE_METADATA.localhost.type }}"
```

Comment:
Can we reuse `switch_type `which i think should be already available ?

Reply:
hi Prince, switch_type is not available in swss_vars.j2. Orchagent gets it by directly reading CFG_DEVICE_METADATA_TABLE_NAME after starting up. This however cannot be done in orchagent.sh to pass the zmq params to orchagent before it starts up.

Reply:
Will plan to add switch_type to swss_vars.j2 as discussed offline